GameOfThrones/Models/Castle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Data.Entity;
+
+namespace GameOfThrones.Models
+{
+    public class Castle
+    {
+        public int CastleId { get; set; }
+        public string Name { get; set; }
+    }
+
+    public class House
+    {
+        public int HouseId { get; set; }
+        public string Name { get; set; }
+        public string Region { get; set; }
+        public string Words { get; set; }
+        public Castle CastleId { get; set; }
+    }
+
+    public class Character
+    {
+        public int CharacterId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public bool Alive { get; set; }
+        public House HouseId { get; set; }
+    }
```

Comment:
Here there is a file called `Castle` in `Models` which implements more than just Castle. It would be best for each distinct class to be implemented in its own separate file.

Reply:
Classes have been separated into their own files